travis.yml

```diff
@@ -0,0 +1,16 @@
+language: python
```

Comment:
@miketsukerman,
requirements.txt is fine but I will only accept a Github Actions configuration file, not Travis.

I had plans to do a Github actions config myself but I assumed that I first needed to make a proper release of cfile module into pypi for the build pipeline to install dependencies correctly. I still haven't had the time to complete the PYPI release of cfile.

Reply:
I see, let me check what is required